### ISSUE TITLE
Decimal separator error in mixed locales

### ISSUE THREAD

**Issue by mrwatson-de** [mrwatson-de/myMateJSON#6]
Script "JSONSetElement - Create expression" generates an invalid expression for fractional numbers for non-European users – those who use a period as a decimal separator:  

I think it needs to be changed the following way:

```
		vType = 2 ; Substitute ( vValue ; Left ( 1/2 ; 1 ) ; JSONGetElement ( Get ( SystemLocaleElements ) ; "Num.Decimal" ) )
```

Issue from [Alex Zueiv](https://community.claris.com/en/0050H00000BqKMO?fromEmail=1&s1oid=00Di0000000eyqY&s1nid=0DB0H000000fxYs&s1uid=0050H00000Bq7Fr&s1ext=0&emkind=chatterDirectMessagePostNotification&emtm=1654487741743)

**Comment by mrwatson-de** [comment]
While this is an error in FileMaker Pro - See [Decimal separator incorrect when cast to text with mixed locales](https://community.claris.com/en/s/question/0D53w00005lqavmCAA/-12-decimal-separator-incorrect-when-cast-to-text-with-mixed-locales) - The suggested workaround will work.

**Comment by mrwatson-de** [comment]
Fixed in [Release v0.9.1](https://github.com/mrwatson-de/myMateJSON/releases/tag/v0.9.1)
